### DISCUSSION TITLE
Laravel 9 with Mongodb Login Problem

### DISCUSSION THREAD

**Discussion by alfirus** [jenssegers/laravel-mongodb#2422] [Q&A]
Hi,
I installed fresh Laravel 9, Mongodb, Mongodb driver and laravel-mongodb. After configuration, i run "php artisan migrate" and everything fine. Then tried to register, no problem. However i try to login, nothing happen. If i put wrong username or password, it will show credential error, mean the connection is not a problem. Is some sort like it didn't create a session.

does anyone faced this kind of problem and how to solve it?

If you like to test it, you can clone https://github.com/alfirus/projectmanagement

**Comment by apeisa** [comment]
Hi @alfirus 

I think this was due to this issue https://github.com/jenssegers/laravel-mongodb/issues/2424 and should work if you upgrade Laravel to the latest (9.22.1 or later) version.

**Reply by alfirus** [reply]
Yeah, after install the new version, everything fine. Thanks @apeisa
